src/reducers/breakpoints.js

```diff
@@ -384,4 +385,23 @@ export function getBreakpointsForSource(state: OuterState, sourceId: string) {
   });
 }
 
+export const getHiddenBreakpoints = createSelector(getBreakpoints, function(
+  breakpoints
+) {
+  return breakpoints
+    .valueSeq()
+    .filter(breakpoint => breakpoint.has("hidden"))
+    .map(hiddenBreakpoint => hiddenBreakpoint.get("location"));
+});
+
+export const getHiddenBreakpoint = createSelector(
+  getHiddenBreakpoints,
+  hiddenBreakpoints => {
+    if (hiddenBreakpoints.length) {
+      return hiddenBreakpoints[0];
+    }
+    return null;
+  }
+);
```

Comment:
maybe we're thinking about this the wrong way. If we're only going to have one temporary breakpoint, perhaps we want a different breakpoint field in the reducer like `pendingBreakpoints`.

I propose we add a `steppingBreakpoint` which does not appear in the UI... do you think that would be easy? we also don't need a pending breakpoint for this case.